### ISSUE TITLE
【小白求助】Novice Seeking Help:Demo 可运行但结果与论文差异大，求复现成功的大佬分享 pip list（ Demo Runs but Results Differ Greatly from Paper, Seeking pip list from Successful Reproducers）小白求助：与论文结果相差较大的原因

### ISSUE THREAD

**Issue by SteinsGateSg** [lxtGH/OMG-Seg#76]
# OMG-LLaVA seek for help
# OMG-LLaVA 结果求助

大家好～我是刚接触这个项目的小白，目前遇到了一个问题想向各位大佬求助。
    
我已成功运行项目中的 gradio demo，但实际输出结果与论文中公布的结果相差较大，关键指标（如准确率、误差值等，可根据项目补充具体指标）未能匹配。
经过初步排查，我推测问题可能出在依赖库版本不兼容上可能导致计算逻辑或结果精度差异。
想恳请已经成功复现论文结果的大佬，分享一下你们环境中的pip list输出内容，帮助我排查版本问题。如果有其他可能导致结果差异的建议，也欢迎随时指出～

非常感谢大家的帮助！

English Text
Hi everyone, I'm a novice new to this project and I'm encountering an issue that I hope to get help with from experts here.

I have successfully run the Demo in the project, but the actual output results differ greatly from those published in the paper. Key metrics (e.g., accuracy, error value, please supplement specific metrics based on the project) fail to match.
After preliminary investigation, I speculate that the problem may be caused by incompatible dependency library versions. Different versions of core libraries (such as PyTorch, TensorFlow, numpy, etc.) may lead to differences in calculation logic or result accuracy.
I would greatly appreciate it if any expert who has successfully reproduced the paper's results could share the output of pip list in your environment to help me troubleshoot version issues. If you have other suggestions that may cause result differences, please feel free to point them out as well!
Thank you very much for your help!
```plaintext
accelerate                    0.26.0
addict                        2.4.0
aiofiles                      23.2.1
aiohappyeyeballs              2.6.1
aiohttp                       3.13.0
aiosignal                     1.4.0
altair                        5.5.0
annotated-types               0.7.0
anyio                         4.11.0
argon2-cffi                   25.1.0
argon2-cffi-bindings          25.1.0
arrow                         1.3.0
arxiv                         2.2.0
asttokens                     3.0.0
async-lru                     2.0.5
async-timeout                 5.0.1
attrs                         25.4.0
babel                         2.17.0
beautifulsoup4                4.14.2
bitsandbytes                  0.45.5
bleach                        6.2.0
blinker                       1.9.0
Brotli                        1.1.0
cachetools                    6.2.0
certifi                       2022.12.7
cffi                          2.0.0
charset-normalizer            2.1.1
click                         8.3.0
colorama                      0.4.6
comm                          0.2.3
contourpy                     1.3.2
cycler                        0.12.1
datasets                      4.1.1
debugpy                       1.8.17
decorator                     5.2.1
deepspeed                     0.17.6
defusedxml                    0.7.1
dill                          0.4.0
distro                        1.9.0
duckduckgo_search             5.3.1b1
einops                        0.8.1
et_xmlfile                    2.0.0
exceptiongroup                1.3.0
executing                     2.2.1
fastapi                       0.118.0
fastjsonschema                2.21.2
feedparser                    6.0.12
ffmpy                         0.6.1
filelock                      3.13.1
fonttools                     4.60.1
fqdn                          1.5.1
frozenlist                    1.8.0
fsspec                        2024.6.1
ftfy                          6.3.1
func_timeout                  4.3.5
gitdb                         4.0.12
GitPython                     3.1.45
gradio                        4.37.2
gradio_client                 1.0.2
gradio_image_prompter         0.1.0
griffe                        0.49.0
h11                           0.16.0
h2                            4.3.0
hf-xet                        1.1.10
hjson                         3.1.0
hpack                         4.1.0
httpcore                      1.0.9
httpx                         0.28.1
huggingface-hub               0.35.3
hyperframe                    6.1.0
idna                          3.4
imageio                       2.37.0
importlib_metadata            8.7.0
importlib_resources           6.5.2
ipykernel                     6.30.1
ipython                       8.37.0
ipywidgets                    8.1.7
isoduration                   20.11.0
jedi                          0.19.2
Jinja2                        3.1.4
json5                         0.12.1
jsonpointer                   3.0.0
jsonschema                    4.25.1
jsonschema-specifications     2025.9.1
jupyter                       1.1.1
jupyter_client                8.6.3
jupyter-console               6.6.3
jupyter_core                  5.8.1
jupyter-events                0.12.0
jupyter-lsp                   2.3.0
jupyter_server                2.17.0
jupyter_server_terminals      0.5.3
jupyterlab                    4.4.9
jupyterlab_pygments           0.3.0
jupyterlab_server             2.27.3
jupyterlab_widgets            3.0.15
kiwisolver                    1.4.9
kornia                        0.8.1
kornia_rs                     0.1.9
lagent                        0.2.4
lark                          1.3.0
lazy_loader                   0.4
Markdown                      3.9
markdown-it-py                4.0.0
MarkupSafe                    2.1.5
mat4py                        0.6.0
matplotlib                    3.10.6
matplotlib-inline             0.1.7
mdurl                         0.1.2
mistune                       3.1.4
mmcv                          2.0.1
mmdet                         3.1.0
mmengine                      0.10.7
mmpretrain                    1.0.1
mmsegmentation                1.1.1
model-index                   0.1.11
modelindex                    0.0.2
modelscope                    1.30.0
mpi4py_mpich                  3.1.5
mpmath                        1.3.0
msgpack                       1.1.1
multidict                     6.7.0
multiprocess                  0.70.16
narwhals                      2.7.0
nbclient                      0.10.2
nbconvert                     7.16.6
nbformat                      5.10.4
nest-asyncio                  1.6.0
networkx                      3.3
ninja                         1.13.0
notebook                      7.4.7
notebook_shim                 0.2.4
numpy                         1.26.4
nvidia-ml-py                  13.580.82
opencv-python                 4.8.1.78
openpyxl                      3.1.5
ordered-set                   4.1.0
orjson                        3.11.3
overrides                     7.7.0
packaging                     25.0
pandas                        2.3.3
pandocfilters                 1.5.1
parso                         0.8.5
peft                          0.7.1
pexpect                       4.9.0
phx-class-registry            4.1.0
pillow                        10.4.0
pip                           25.2
platformdirs                  4.4.0
prettytable                   3.16.0
prometheus_client             0.23.1
prompt_toolkit                3.0.52
propcache                     0.4.0
protobuf                      6.32.1
psutil                        7.1.0
ptyprocess                    0.7.0
pure_eval                     0.2.3
py-cpuinfo                    9.0.0
pyarrow                       21.0.0
pycocotools                   2.0.10
pycparser                     2.23
pydantic                      2.10.6
pydantic_core                 2.27.2
pydeck                        0.9.1
pydub                         0.25.1
Pygments                      2.19.2
pyparsing                     3.2.5
python-dateutil               2.9.0.post0
python-json-logger            4.0.0
python-multipart              0.0.20
pytz                          2025.2
PyYAML                        6.0.3
pyzmq                         27.1.0
referencing                   0.36.2
regex                         2025.9.18
requests                      2.32.5
rfc3339-validator             0.1.4
rfc3986-validator             0.1.1
rfc3987-syntax                1.1.0
rich                          14.1.0
rpds-py                       0.27.1
ruff                          0.13.3
safetensors                   0.6.2
scikit-image                  0.25.2
scipy                         1.15.3
semantic-version              2.10.0
Send2Trash                    1.8.3
sentencepiece                 0.2.1
setuptools                    78.1.1
sgmllib3k                     1.0.0
shapely                       2.1.2
shellingham                   1.5.4
six                           1.17.0
smmap                         5.0.2
sniffio                       1.3.1
socksio                       1.0.0
soupsieve                     2.8
stack-data                    0.6.3
starlette                     0.48.0
streamlit                     1.50.0
sympy                         1.13.3
tenacity                      9.1.2
termcolor                     3.1.0
terminado                     0.18.1
terminaltables                3.1.10
tifffile                      2025.5.10
tiktoken                      0.12.0
timeout-decorator             0.5.0
timm                          1.0.20
tinycss2                      1.4.0
tokenizers                    0.15.2
toml                          0.10.2
tomli                         2.2.1
tomlkit                       0.12.0
torch                         2.1.2+cu118
torchaudio                    2.1.2+cu118
torchvision                   0.16.2+cu118
tornado                       6.5.2
tqdm                          4.67.1
traitlets                     5.14.3
transformers                  4.36.0
transformers-stream-generator 0.0.5
triton                        2.1.0
typer                         0.19.2
types-python-dateutil         2.9.0.20250822
typing_extensions             4.12.2
typing-inspection             0.4.2
tzdata                        2025.2
uri-template                  1.3.0
urllib3                       2.5.0
uvicorn                       0.37.0
watchdog                      6.0.0
wcwidth                       0.2.14
webcolors                     24.11.1
webencodings                  0.5.1
websocket-client              1.8.0
websockets                    11.0.3
wheel                         0.45.1
widgetsnbextension            4.0.14
xtuner                        0.1.21         /root/autodl-tmp/OMG-Seg-main/OMG-Seg-main/omg_llava
xxhash                        3.6.0
yapf                          0.43.0
yarl                          1.22.0
zipp                          3.23.0
```
![](https://images.cnblogs.com/cnblogs_com/blogs/828475/galleries/2447233/o_251015055209_%E5%B1%8F%E5%B9%95%E6%88%AA%E5%9B%BE%202025-10-15%20135158.png)
